controller/fileStore.go

```diff
@@ -85,7 +85,11 @@ func (fs *FileStore) fileStoreService() {
 		case WRITE:
 			response.error = ioutil.WriteFile(path.Join(fs.root, req.fileName), req.fileContents, 0600)
 		case DELETE:
-			response.error = os.Remove(path.Join(fs.root, req.fileName))
+			response.error = nil
+			filePath := path.Join(fs.root, req.fileName)
+			if _, err := os.Stat(filePath); os.IsExist(err) {
```

Comment:
Won't err just be nil if the stat succeeds?  I would suggest unconditionally doing the `os.Remove` but checking for and ignoring err when `os.IsNotExist(err)`.